features/mbedtls/importer/adjust-config.sh

```diff
@@ -56,8 +68,8 @@ add_code
     "\n"                                                                                          \
     "#else\n"
 
-add_code                                                                                                       \
-    "#include \"mbedtls\/check_config.h\"\n"                                                                   \
+prepend_code                                                                                                   \
+    "#endif \/\* MBEDTLS_CONFIG_H \*\/"                                                                        \
```

Comment:
This new approach only makes sense if Mbed TLS has changed to not include `check_config.h` in `config.h`, which hasn't happened yet. How about changing `adjust-config.sh` to work both with the current Mbed TLS and with a future Mbed TLS without `check_config.h` in `config.h`, by replacing at `#include MBEDTLS_USER_CONFIG_FILE`?
``

Reply:
As I said above:

> I recommend to the maintainers that they put in their backlog to remove or replace these scripts. They were added as a temporary measure some years ago, and were never updated.

The correct approach is to remove the script and do this differently, preferably without scripts that adapt the source code.

Currently these scripts are brittle and break easily, and this is a modest change to address the immediate problem only.